src/pages/call/calling.ts

```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+import { NavController, NavParams } from 'ionic-angular';
+import { UserData } from '../../providers/user-data';
+
+@Component({
+  selector: 'page-calling',
+  templateUrl: 'calling.html',
+})
+export class CallingPage {
+    phone_number: string = '';
+
+    constructor(public navCtrl: NavController,
+                public navParams: NavParams,
+                public userData: UserData) {
+       this.phone_number = this.navParams.get('phone_number');
```

Comment:
@jordibc why don't you create a form to submit the phone number?  We are doing it [here](https://github.com/EyeSeeTea/FIRE-WiFiCalling/blob/feature/voucher-page/src/pages/topup/addVoucher/addVoucher.html). I don't have a strong opinion about this. Maybe we want to do some kind of validations (right phone format?) and a form is a better option. What do u think?